DuckDuckGo/VPNFeedbackForm/VPNMetadataCollector.swift

```diff
@@ -329,3 +329,13 @@ final class DefaultVPNMetadataCollector: VPNMetadataCollector {
     }
 
 }
+
+// MARK: - Unified feedback form support
+
+extension VPNMetadata: UnifiedFeedbackMetadata {}
+
+extension DefaultVPNMetadataCollector: UnifiedMetadataCollector {
+    func collectMetadata() async -> VPNMetadata? {
+        await collectVPNMetadata()
+    }
```

Comment:
Does this need to return optional? Looks like `collectVPNMetadata()` doesn't return optional, so maybe we can simplify.